packages/dashboard/.env.testnet.example

```diff
@@ -0,0 +1,9 @@
+SIDETREE_METHOD="example:sidetree.testnet"
```

Comment:
perhapse name this file less ambigiously... "example:sidetree.testnet" or similar.

Reply:
Updated PR!